datasets/oclar/README.md

```diff
@@ -0,0 +1,177 @@
+---
+annotations_creators:
+
+- crowdsourced language_creators:
+- crowdsourced languages:
+- ar licenses:
+- unknown multilinguality:
+- monolingual size_categories:
+- 1K<n<10K source_datasets:
+- original task_categories:
+- text-classification
+- text-scoring task_ids:
+- sentiment-classification
+- sentiment-scoring
+
```

Comment:
The format is a bit weird here. Maybe an issue when copy-pasting the yaml from the app ?

you can find information about how to create the tags here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Reply:
![image](https://user-images.githubusercontent.com/26907161/101543265-4af63000-39a4-11eb-9d58-de7ac412f640.png)

Reply:
indeed, it's pycharm's code reformating 
my bad